aws/provider_test.go

```diff
@@ -1269,6 +1269,81 @@ func TestAccAWSProvider_IgnoreTags_Keys_Multiple(t *testing.T) {
 	})
 }
 
+func TestAccProvider_DefaultTags_EmptyConfigurationBlock(t *testing.T) {
```

Comment:
Nit: Looks like the other tests in this file are currently using the `TestAccAWSProvider_` prefix so might be worth keeping that for consistency for now so there's no missed testing runs.